com.endlessm.apps.Sdk.json.in.in

```diff
@@ -512,6 +512,16 @@
                 }
             ]
         },
+        {
+            "name": "eos-knowledge-content-renderer",
```

Comment:
I think it would be good to `cleanup` the header files from this build so that it's clear to downstream users of the SDK that it's not an API that they are supposed to use.

Reply:
This comment still needs addressing.

Reply:
Oops, didn't end up pushing the right branch (HEAD was still detached locally, grumble). Pushed now